torchgeo/trainers/segmentation.py

```diff
@@ -250,9 +349,13 @@ def validation_step(
         batch_size = x.shape[0]
         y_hat = self(x)
         loss = self.criterion(y_hat, y)
-        self.log('val_loss', loss, batch_size=batch_size)
+        self.log('val_loss', loss, batch_size=batch_size, on_epoch=True)
```

Comment:
I believe `on_epoch=True` is the default behavior for `validation_step` and `training_step` (even though the latter is [undocumented](https://lightning.ai/docs/pytorch/stable/extensions/logging.html#logging-from-a-lightningmodule)), so maybe we could get rid of this?